app/models/manageiq/providers/openstack/cloud_manager/orchestration_template.rb

```diff
@@ -63,7 +77,7 @@ def deployment_options(_manager_class = nil)
       :description => "Abort the creation if it does not complete in a proper time window",
     )
 
-    super << onfailure_opt << timeout_opt
+    [tenant_opt] + super << onfailure_opt << timeout_opt
```

Comment:
@bzwei any reason to not just do `super << onfailure_opt << timeout_opt << tenant_opt` ?

Reply:
The `tenant_opt` is the first option on the dialog for backward compatibility purpose.